### PR TITLE
Fix/bucket create

### PR DESCRIPTION
**Pull request by joshLong145**
- fix to bucket RPC commands panic in certain cases.
## Changes

- Item 1
- Item 2

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: internal/binding/binding.go [hunk 1]
```diff
@@ -49,7 +49,9 @@ Initialize the speedway binding to the motor
 */
 func InitMotor() mtr.MotorNode {
 	initreq := &rtmv1.InitializeRequest{
-		DeviceId: utils.GetHwid(),
+		DeviceId:   utils.GetHwid(),
+		LogLevel:   "warn",
+		ClientMode: 0,
```
Comment (mcjcloud) [inline internal/binding/binding.go:54]: Can you use the contants here instead of integer?

File: internal/daemon/bucket.go [hunk 2]
```diff
@@ -1,42 +1,57 @@
...
+		WhereIs: res.WhereIs,
 	}
+
 	return nil
 }
 
 func (d *Daemon) GetBucketsByCreator(req rtmv1.QueryWhereIsByCreatorRequest, res *rtmv1.QueryWhereIsByCreatorResponse) (err error) {
 	r, e := d.instance.Instance.QueryWhereIsByCreator(req)
-	*res = *r
+	*res = rtmv1.QueryWhereIsByCreatorResponse{
```
Comment (mcjcloud) [inline internal/daemon/bucket.go:23]: Why is it that you need to unpack each property like this?

**Review by mcjcloud — CHANGES_REQUESTED** [review]

File: internal/daemon/bucket.go (hunk 2 above)
Comment (joshLong145) [reply internal/daemon/bucket.go:23]: I was observing weird error with derlerencing the `r` variable and and assigning it to a new pointer. was observing cases where the `res` would be empty once returned to the client. this unpacking keeps panics from occurring.

File: internal/binding/binding.go (hunk 1 above)
Comment (joshLong145) [reply internal/binding/binding.go:54]: ah good point, will change this.

File: internal/daemon/bucket.go (hunk 2 above)
Comment (mcjcloud) [reply internal/daemon/bucket.go:23]: Let's instead add a nil check to `r` so as to avoid avoid needing to update this code with every proto update.

**Review by mcjcloud — APPROVED** [review]

**Pull request merged**